### PR TITLE
Use Optional for qubit parameters

### PR DESCRIPTION
**Pull request by xbe**

### CODE REVIEW

File: src/lib/state.py [hunk 1]
```diff
@@ -148,7 +149,7 @@ def apply_controlled(self, gate, control, target) -> None:
 # be distinguished when multiplied with an arbitrary complex number, aka,
 # global phase.
 #
-def qubit(alpha:float=None, beta:float=None) -> State:
+def qubit(alpha: Optional[float]=None, beta: Optional[float]=None) -> State:
```
Comment (qcc4cp) [inline src/lib/state.py:152]: Now that I'm looking at this... alpha and beta should be complex numbers. Perhaps replace with np.complexfloating?

Comment (xbe) [reply src/lib/state.py]: Done.

**Review by qcc4cp — COMMENTED** [review]
Perfect, Thanks

**Comment by qcc4cp** [comment]
Thanks again

**Pull request merged**